content/providers/05-observability/langsmith.mdx

```diff
@@ -205,17 +205,19 @@ await generateText({
 ### Customize run ID
 
 You can customize the run ID by passing the `runId` argument to the `AISDKExporter.getSettings()` method. This is especially useful if you want to know the run ID before the run has been completed.
+Note that the run ID has to be a valid UUID.
```

Comment:
Can you use `<Note>...</Note>` here? (pls make sure to run `pnpm prettier-fix` afterwards)?